media-libs/openh264/openh264-2.1.0.ebuild

```diff
@@ -0,0 +1,105 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils multilib-minimal toolchain-funcs
```

Comment:
Why do we still need `eutils` eclass?

Reply:
While there please ensure that `multilib-minimal` eclass will come last, see https://gitweb.gentoo.org/repo/gentoo.git/tree/eclass/multilib-minimal.eclass?id=e5005d5f96c26cad463c38c1c7cdd4a79f218938#n16

Reply:
> Why do we still need `eutils` eclass?
usex?

Reply:
> While there please ensure that `multilib-minimal` eclass will come last, see https://gitweb.gentoo.org/repo/gentoo.git/tree/eclass/multilib-minimal.eclass?id=e5005d5f96c26cad463c38c1c7cdd4a79f218938#n16

Fixed

Reply:
> > Why do we still need `eutils` eclass?
> > usex?

OK but it isn't needed since EAPI 5 anymore. Please remove `eutils` eclass from `inherit`.